resolvedvelocities/synthetic_data/Field.py

```diff
@@ -0,0 +1,76 @@
+# Field.py
+import numpy as np
+from apexpy import Apex
+from scipy import interpolate
+import coord_convert as cc
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+
+import configparser
+
+class Field(object):
+
+    def __init__(self, config):
+
+        self.read_config(config)
+
+        # convert list from config file to arrays
+        self.field_coords = np.array(self.field_coords)
+        self.field_values = np.array(self.field_values)
+
+        # initialize Apex object
+        self.A = Apex(date=self.apex_year)
```

Comment:
It might be clearer to read in the code if we used:

`self.apex = Apex()`

What do you think?